### PR TITLE
[2/2] Install and run agent

### PR DESCRIPTION
**Pull request by adamstegman**
Installs and runs the datadog agent.

The agent runs on Python, and installs dependencies with pip. To shortcut a bunch more work, I'm using the `python-dev` Ubuntu package, so this will only work on Ubuntu stemcells.

I also skipped setting up `jmxfetch` because I didn't want to also install Java. Can always add it as a second job.

### CODE REVIEW

File: jobs/datadog-agent/templates/helpers/ctl_setup.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
```
Comment (adamstegman) [inline jobs/datadog-agent/templates/helpers/ctl_setup.sh]: boilerplate, I did not modify this from [bosh-gen](https://github.com/cloudfoundry-community/bosh-gen)

File: jobs/datadog-agent/templates/helpers/ctl_utils.sh [hunk 2]
```diff
@@ -0,0 +1,155 @@
+# Helper functions used by ctl scripts
```
Comment (adamstegman) [inline jobs/datadog-agent/templates/helpers/ctl_utils.sh]: boilerplate, I did not modify this from [bosh-gen](https://github.com/cloudfoundry-community/bosh-gen)

File: src/apt/fetch_debs.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
+set -e          # exit immediately if a simple command exits with a non-zero status
```
Comment (adamstegman) [inline src/apt/fetch_debs.sh]: boilerplate, I did not modify this from [bosh-gen](https://github.com/cloudfoundry-community/bosh-gen)

**Comment by jeremiemv** [comment]
A little more complex than the New Relic Agent release...

**Comment by adamstegman** [comment]
> A little more complex than the New Relic Agent release...

Python! 💯

**Comment by mitch-lindsay** [comment]
image
:shipit:

**Pull request merged**